Test/Integration/AssertValues/Magento247.php

```diff
@@ -4,7 +4,7 @@
 
 class Magento247
 {
-    public $productsOnStockCount = 182;
+    public $productsOnStockCount = 180;
     public $productsOutOfStockCount = 183;
```

Comment:
I've observed that this is also the expected value on Magento 2.4.6.

Reply:
Yes, this is due to the fact that Giftcards are not indexed when they are out of stock. I'm pretty sure that they were at some point, I guess this will require some investigation in MAGE-1076 once we have time.

Reply:
Hmm - yeah that's a weird one. Still when I ran the same tests against 2.4.6, I was getting failures on these assertion values. I would expect consistent outcomes on the tests since we set up the test case.

Reply:
Just an idea, how about creating a Magento24 class and then both Magento246 and Magento247 extending from Magento24? In that way, we can keep common stuffs on super class.

Reply:
Makes sense ! TBH I'm not even sure that there are siginificant changes between minor release, I didn't test the values on 2.4.6 but they must be the same.